2/contrib/openshift/base-plugins.txt

```diff
@@ -1,9 +1,10 @@
 ant:1.11
 blueocean:1.24.8
 config-file-provider:3.8.1
-configuration-as-code:1.52
+configuration-as-code:1.55.1
 configuration-as-code-groovy:1.1
-credentials-binding:1.27
+credentials-binding:1.27.1
+cloudbees-bitbucket-branch-source:746.v350d2781c184
```

Comment:
This is the right call here. This plugin is a transitive dep of `blueocean` - they haven't cut a release that updates their dependency. I don't think there is a major risk that this breaks blueocean.